config/locales/en.yml

```diff
@@ -64,7 +64,7 @@ en:
               invalid: Enter an email address in the correct format, like name@example.com
   application_choice:
     status_name:
-      application_complete: 'Application completed'
+      application_complete: 'New'
```

Comment:
just it sounds a bit weird to have 
`application_complete: 'New'`

Reply:
This is a conflict between the ProVendor and candidate perspectives. We could translate differently in each context, but I think it's better to have one word for this state instead of two — `Completed` could do the job?

Reply:
Let's revisit this later, as the status name is currently only used for providers. Maybe we'll translate the states differently for candidates & others.